css/properties/-webkit-touch-callout.json

```diff
@@ -30,7 +30,7 @@
               "version_added": false
             },
             "opera_android": {
-              "version_added": true
+              "version_added": false
```

Comment:
Thanks for catching that, I must’ve somehow missed it when copying from my template.